app/src/db/migrations/20240327213908_bring-forward-feat.ts

```diff
@@ -0,0 +1,31 @@
+import type { Knex } from 'knex';
```

Comment:
Let's rename this to file to `20240327000000_001-bring-forward.ts`. I should've followed our historical naming convention with the init migration, but it slipped by before going to prod. So let's start this now.